ipt_connect/IPTdev/templates/IPTdev/rounds.html

```diff
@@ -45,21 +39,22 @@ <h1>{% trans 'Rounds overview' %}</h1>
         <table>
             <tr>
                 <!--
-                TODO: detablize this caption
+                TODO: detablize?
```

Comment:
I think you can simply remove this todo...don't know why I put it here (if I'm the one who did that...)

Reply:
It was me. Using `<table>`s  for things which are not tables indeed, isn't good, is it? Here we had a `<table>` 2x3, now wehave 2x1, since it is still not a table :)